prow/plugins/config.go

```diff
@@ -420,6 +422,9 @@ type ConfigMapSpec struct {
 	GZIP *bool `json:"gzip,omitempty"`
 	// Namespaces is the fully resolved list of Namespaces to deploy the ConfigMap in
 	Namespaces []string `json:"-"`
+	// Clusters is a map from cluster to namespaces
+	// which specifies the targets the configMap needs to be deployed, i.e., each namespace in map[cluster]
+	Clusters map[string][]string `json:"clusters"`
```

Comment:
With this approach we should deprecate `AdditionalNamespaces` and `Namespace` and use those fields to populate `Clusters[DefaultCluster]`, right?

Reply:
Ah, I see you are already doing this -- let's add the deprecation notice? Or do we want to allow people to always use the original fields?